src/bmi_cfe.c

```diff
@@ -213,38 +229,53 @@ static const char *output_var_locations[OUTPUT_VAR_NAME_COUNT] = {
         "node",
         "node",
         "node",
-        "node"
+        "node",
+	"domain",
```

Comment:
These should all be "node".  See https://bmi.readthedocs.io/en/latest/#get-var-location

Reply:
okay, I changed the "domain" to "node", however, these variables are not defined on nodes, they are integrated or computed quantities based on variables defined on nodes.